vxdetector/Output_counter.py

```diff
@@ -86,41 +98,32 @@ def create_output(path, file_name, unaligned_count, dictionary, dir_name,
         writer = csv.DictWriter(o, fieldnames=fieldnames)
         if header is False:
             writer.writeheader()
+        for key in dictionary:
+            dictionary[key] = round(dictionary[key], 2)
         writer.writerow({'Read-file': file_name, 'Number of Reads': all_reads,
```

Comment:
since you now work with a global variable, we should avoid manual iteration of keys here. How about
```
foo = {'Read-file': file_name, 'Number of Reads': all_reads, ...}
foo.update(dictionary)
```